test/source/tests/decrypt.ts

```diff
@@ -1158,6 +1158,7 @@ XZ8r4OC6sguP/yozWlkG+7dDxsgKQVBENeG6Lw==
           await printPage.waitForContent('@print-to', 'To: flowcrypt.compatibility@gmail.com');
           await printPage.waitForContent('@print-cc', 'ci.tests.gmail@flowcrypt.dev');
           await printPage.waitForContent('@print-content', 'Test print message');
+          expect(!isNaN(Date.parse(await printPage.readHtml('@print-date')))).to.equal(true);
```

Comment:
here you just check that `print-date` contains valid date, but don't check it's value.
let's also check for correct string - parse sent date of message with id `18ecbf57e1dfb9b5` and check if it's `.toLocaleString()` value equals to `print-date` value:
```ts
const expectedPrintDateString = (new Date(1712818847000)).toLocaleString();
await printPage.waitForContent('@print-content', expectedPrintDateString);
```

Reply:
Hello @sosnovsky, I've done similar with this and the reason I have updated it with the current one is that. The locale that the test environment and the locale for our local host machine are different so we've got different results. I honestly not sure how to achieve it for both it will have same result for CI and local host environment - I'm sorry.

Reply:
But in mentioned code we convert unix timestamp `1712818847000` to locale string on the running machine (which simulates behavior of our browser extension), so it should give correct results in both environments - locally and when running CI tests. Can you please re-try it? Thanks